### PR TITLE
Devnet update

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**

### CODE REVIEW

**Review by cor — APPROVED** [review]
LGTM, let's hope the deploy goes well

**Comment by hussein-aitlahcen** [comment]
@cor finally passing, are we ok to merge it? If so I'm going to do a final rebase

**Comment by cor** [comment]
@hussein-aitlahcen you have my approval, time for the final rebase :-)

**Review by andor0 — APPROVED** [review]

**Pull request merged**
